mobile/lib/services/secure_storage.dart

```diff
@@ -9,11 +9,24 @@ class SecureStorage {
   SecureStorage._internal();
   static final SecureStorage _instance = SecureStorage._internal();
 
-  final _secureStorage = const FlutterSecureStorage();
+  final _secureStorage = const FlutterSecureStorage(
+      aOptions: AndroidOptions(
+    encryptedSharedPreferences: true,
+  ));
 
   Future<void> clearUserData() async {
-    const secureStorage = FlutterSecureStorage();
-    await secureStorage.deleteAll();
+    await _secureStorage.deleteAll();
```

Comment:
There is this static analysis check above my comment. It is about a "trailing comma missing". Care to review this further? @NoahNsimbe

Reply:
Thanks @Baalmart